### PR TITLE
brew-bundle, caffeinate, open and tmutil: add German translation

### PR DESCRIPTION
**Pull request by GETandSELECT**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages.de/osx/brew-bundle.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# brew bundle
+
+> Bundler für Homebrew, Homebrew Cask und den Mac App Store.
+> More information: <https://github.com/Homebrew/homebrew-bundle>.
```
Comment (bl-ue) [inline pages.de/osx/brew-bundle.md:4]: More like the other German pages?
```suggestion
> Mehr Informationen: <https://github.com/Homebrew/homebrew-bundle>.
```

File: pages.de/osx/caffeinate.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# caffeinate
+
+> Hindert den Mac daran in den Schlaf-Modus zu gehen
```
Comment (bl-ue) [inline pages.de/osx/caffeinate.md:3]: Missing period?
```suggestion
> Hindert den Mac daran in den Schlaf-Modus zu gehen.
```

File: pages.de/osx/tmutil.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
+# tmutil
+
+> Dienstprogramm zum Verwalten von Time Machine-Backups. Die meisten Kommandos erfordern Root-Rechte.
+> More information: <https://ss64.com/osx/tmutil.html>.
+
+- Setze ein HFS+ Laufwerk als Backupziel::
```
Comment (bl-ue) [inline pages.de/osx/tmutil.md:6]: Duplicate colon?
```suggestion
- Setze ein HFS+ Laufwerk als Backupziel:
```

File: pages.de/osx/tmutil.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+- Aktivieren von automatischen Backups:
+
+`sudo tmutil enable`
+
+- Automatische Backups deaktivieren:
+
+`sudo tmutil disable`
+
+- Starte ein Backup im Hintergrund, falls nicht bereits eines läuft
```
Comment (bl-ue) [inline pages.de/osx/tmutil.md:26]: Missing colon?
```suggestion
- Starte ein Backup im Hintergrund, falls nicht bereits eines läuft:
```

File: pages.de/osx/tmutil.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+`sudo tmutil startbackup`
+
+- Starte ein Backup im Vordergrund, falls nicht bereits eines läuft
```
Comment (bl-ue) [inline pages.de/osx/tmutil.md:30]: Missing colon?
```suggestion
- Starte ein Backup im Vordergrund, falls nicht bereits eines läuft:
```

**Review by bl-ue — CHANGES_REQUESTED** [review]
Hi @GETandSELECT, thank you for all the translations!  I've left some comments below for you to review.

File: pages.de/osx/tmutil.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
+# tmutil
+
+> Dienstprogramm zum Verwalten von Time Machine-Backups. Die meisten Kommandos erfordern Root-Rechte.
+> More information: <https://ss64.com/osx/tmutil.html>.
```
Comment (bl-ue) [inline pages.de/osx/tmutil.md:4]:
```suggestion
> Mehr Informationen: <https://ss64.com/osx/tmutil.html>.
```

**Review by schneiderl — APPROVED** [review]
Don't know the language but looks good to me.

**Comment by GETandSELECT** [comment]
thanks @bl-ue 
last PR the CICD pipeline detected missing "." and ":"
Sorry for those

**Comment by bl-ue** [comment]
>last PR the CICD pipeline detected missing "." and ":"?

That's because the CI/CD uses [tldr-bot](https://github.com/tldr-pages/tldr-bot) which in turn uses [tldr-lint](https://github.com/tldr-pages/tldr-lint), which, very unfortunately, does not support non-English pages.  Eventually, we might be able to fix that, but in the meantime, I have a few scripts I use to check basic syntax.

**Review by bl-ue — APPROVED** [review]

**Comment by bl-ue** [comment]
@GETandSELECT in the future, will you please create a new branch, based off of master, when making a PR?  Because since you use `master` on your fork for each PR, it can sometimes make reviewing locally just a tad bit harder :)

**Pull request merged**